jest.config.js

```diff
@@ -9,7 +9,7 @@ module.exports = {
       statements: 100,
     },
   },
-  moduleFileExtensions: ['ts', 'tsx', 'json', 'js', 'jsx', 'node'],
+  moduleFileExtensions: ['ts', 'json', 'js', 'node'],
```

Comment:
Note that this is for the file extensions used not just by the project directly, but by _all dependencies_. [The Jest docs suggest moving the most commonly-used extensions to the left](https://jestjs.io/docs/configuration#modulefileextensions-arraystring), but I'd suggest leaving all of the default entries there.

I discovered this when I went to remove `node` from the array on the `@metamask/controllers` repo, and the tests broke! Apparently a dependency is using the `.node` file extension.